src/components/namestable/worker-edit.component.tsx

```diff
@@ -109,7 +110,8 @@ export function WorkerEditComponent(options: WorkerEditComponentOptions): JSX.El
 
   const updateWorkerInfoBatch = useCallback(
     (newStatePart: Partial<WorkerInfoExtendedInterface>): void => {
-      setWorkerInfo((prev) => ({ ...prev, ...newStatePart }));
+      const cleanedObject = pickBy(newStatePart, (v) => v !== undefined);
```

Comment:
Wczesniej było tak, że pod czas edycji pracownika dane o nim były pobierany ze stanu na podstawie imienia.
W przypadku jeśli takiego pracownika wcześniej nie było, dane o etacie i stanowisku były usuwane (ponieważ nie było tego w stanie i te pola w obiekcie miały wartość undefined)
Poprawiłem to i zrobiłem tak, ze do stanu są dodawane tylko te pola co mają jakoś wartość